### PR TITLE
Adding feature for checking internet connection status

### PR DESCRIPTION
**Pull request by stulees**
- Updating npm dependencies and fixing CCI build
- Adding `hasInternetConnection` function to common module
- Uses a provided file on a server, makes a HEAD request and executes the provided callback function with the result
- Unit tests added
- Feature version bump

### CODE REVIEW

**Comment by stulees** [comment]
@rodrigopavezi Figured out the build problem, package.json was missing dependencies `bower` and `phantomjs`. Please review, thanks

File: circle.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
+machine:
+  node:
+    version: 4.2.2
+  environment:
+    awscli: $HOME/aws/bin/aws
```
Comment (rodrigopavezi) [inline circle.yml]: @stulees I think we don't need this as it is not deployed to amazon. right? cheers

File: circle.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 dependencies:
+  cache_directories:
+    - /home/ubuntu/aws
```
Comment (rodrigopavezi) [inline circle.yml]: the same for this one.

File: dist/all.js [hunk 3]
```diff
@@ -332,6 +332,25 @@ RiseVision.Common.Utilities = (function() {
...
+    if (!filePath || !callback || typeof callback !== "function") {
+      return;
+    }
+
+    xhr.open("HEAD", filePath + "?cb=" + new Date().getTime(), false);
+
+    try {
+      xhr.send();
+
+      callback((xhr.status >= 200 && xhr.status < 304));
```
Comment (rodrigopavezi) [inline dist/all.js]: @stulees I wonder if it should allow 304 too?

Comment (stulees) [inline dist/all.js]: @rodrigopavezi The request will always include a cachebuster, so I'm not sure if allowing 304 is right.

**Comment by stulees** [comment]
@rodrigopavezi Made the changes to `circle.yml`, thanks

File: dist/all.js (hunk 3 above)
Comment (rodrigopavezi) [inline dist/all.js]: @stulees alright, makes sense. cheers

**Comment by rodrigopavezi** [comment]
@stulees looks good :+1: cheers

**Pull request merged**